docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -186,6 +186,10 @@ Noteworthy module changes
 
 * The ``docker_swarm_service`` module no longer sets a default for the ``user`` option. Before, the default was ``root``.
 
+* The ``docker_swarm_service`` module no longer sets a default for the ``update_delay`` option. Before, the default was ``10``.
```

Comment:
Do you think it would make sense to combine the `docker_swarm_service` entries? Something like `Several option defaults of the ``docker_swarm_service`` module were removed:` with a sublist of the individual options?

Reply:
Yes that makes sense.